test/tests/functional/pbs_release_limited_res_suspend.py

```diff
@@ -848,3 +848,84 @@ def test_normal_user_unable_to_see_res_released(self):
         self.assertFalse("resource_released_list.mem" in attrs[0],
                          "Normal user can see resources_released_list "
                          "which is not expected")
+
+    def test_if_node_gets_oversubscribed(self):
+        """
+        Check if the node gets oversubscribed if a filler job runs
+        on resources left on the node after suspension.
+        """
+        # Set mem in restrict_res_to_release_on_suspend server attribute
+        a = {ATTR_restrict_res_to_release_on_suspend: 'mem'}
+        self.server.manager(MGR_CMD_SET, SERVER, a)
+
+        a = {ATTR_sched_preempt_enforce_resumption: True}
+        self.server.manager(MGR_CMD_SET, SCHED, a)
+
+        j1 = Job(TEST_USER)
+        j1.set_attributes({ATTR_l + '.select': '1:ncpus=2:mem=512mb'})
+        jid1 = self.server.submit(j1)
+        self.server.expect(JOB, {ATTR_state: 'R'}, id=jid1)
+
+        # Submit a filler job
+        j2 = Job(TEST_USER)
+        j2.set_attributes({ATTR_l + '.select': '1:ncpus=3',
+                           ATTR_l + '.walltime': 50})
+        jid2 = self.server.submit(j2)
+        self.server.expect(JOB, {ATTR_state: 'Q'}, id=jid2)
+
+        # Submit a high priority job
+        j3 = Job(TEST_USER)
+        j3.set_attributes({ATTR_l + '.select': '1:ncpus=1:mem=2gb',
+                           ATTR_q: 'expressq',
+                           ATTR_l + '.walltime': 100})
+        jid3 = self.server.submit(j3)
+        self.server.expect(JOB, {ATTR_state: 'R'}, id=jid3)
+        self.server.expect(JOB, {ATTR_state: 'S'}, id=jid1)
+        # Check that resources_assigned is not exceeding resources_available
```

Comment:
@I don't think it really matters, but jid3 can't be running until jid1 is suspended, you could switch this check around to look for the suspended job before looking for the running job.  But again, I doubt it really matters.

Reply:
You are right Lisa, it would not matter. If you are ok with it, I'd like to retain it as is

Reply:
I am fine to leave it as is.